integration-cli/docker_cli_images_test.go

```diff
@@ -18,6 +18,40 @@ func (s *DockerSuite) TestImagesEnsureImageIsListed(c *check.C) {
 	}
 }
 
+func (s *DockerSuite) TestImagesEnsureImageWithTagIsListed(c *check.C) {
+	_, err := buildImage("imagewithtag:v1",
+		`FROM scratch
+		MAINTAINER dockerio1`, true)
+	c.Assert(err, check.IsNil)
+
+	time.Sleep(time.Second)
```

Comment:
In another PR is fine.
1sec is a lot of time, especially if a lot of tests are doing this sleep.
Thanks for clarifying!